--- source/com/wutka/dtd/Scanner.java.orig	2013-07-27 23:42:55.310125203 +0100
+++ source/com/wutka/dtd/Scanner.java	2013-07-27 23:43:34.668405644 +0100
@@ -502,11 +502,6 @@
 //    sub-section delimiters '<![' and ']]>'. These must be balanced,
 //    but no section keyword is required:
 //    Conditional Section
-//[61] conditionalSect ::=  includeSect | ignoreSect
-//[62] includeSect ::=  '<![' S? 'INCLUDE' S? '[' extSubsetDecl ']]>'
-//[63] ignoreSect ::=  '<![' S? 'IGNORE' S? '[' ignoreSectContents* ']]>'
-//[64] ignoreSectContents ::=  Ignore ('<![' ignoreSectContents ']]>' Ignore)*
-//[65] Ignore ::=  Char* - (Char* ('<![' | ']]>') Char*)
 
         for (;;)
         {
@@ -524,8 +519,6 @@
                     {
                         if ( nestingDepth == 0)
                         {
-                            // The end of the IGNORE conditional section
-                            // has been found.  Break out of for loop.
                             break;
                         }
                         else
@@ -547,8 +540,6 @@
                     ch = read();
                     if ( ch == '[' )
                     {
-                        // The beginning of a new ignoreSectContents section
-                        // has been found.  Increment nesting depth.
                         nestingDepth++;
                     }
                 }
